make/targets/openshift/codegen.mk

```diff
@@ -9,6 +11,7 @@ CODEGEN_OUTPUT_PACKAGE ?=$(error CODEGEN_OUTPUT_PACKAGE is required)
 
 define run-codegen
 $(CODEGEN_PKG)/generate-groups.sh \
+	"$(SHELL)" \
```

Comment:
shouldn't this be one line above? I agree this is needed as after migrating to gomod executable bits get stripped

Reply:
Indeed, I mixed it up when I edited the file directly on GH.

Opened #16 to address that.

/close

Reply:
why didn't you just update this PR? opening a PR per change seems unusual and looses history